src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -354,27 +357,20 @@ protected void customizeCellRenderer(JTable table, @Nullable Object value, boole
   }
 
   public static class EntryFilter {
-    @Nullable
-    private final String text;
-    private final boolean isRegex;
-    private final boolean isMatchCase;
+    @NotNull
+    private final FlutterLogFilterPanel.FilterParam filterParam;
 
-    public EntryFilter(@Nullable String text) {
-      this(text, false, false);
-    }
 
-    public EntryFilter(@Nullable String text, boolean isMatchCase, boolean isRegex) {
-      this.text = text;
-      this.isMatchCase = isMatchCase;
-      this.isRegex = isRegex;
-    }
-
-    @Nullable
-    public String getText() {
-      return text;
+    public EntryFilter(@NotNull FlutterLogFilterPanel.FilterParam filterParam) {
+      this.filterParam = filterParam;
     }
 
     public boolean accept(@NotNull FlutterLogEntry entry) {
+      if (entry.getLevel() < filterParam.getLogLevel().value) {
```

Comment:
For filter log by level, I simply check `entry.getLevel() < filterParam.getLogLevel().value`.
Because we define:
```java
  public enum Level {
    NONE(0),
    FINEST(300),
    FINER(400),
    FINE(500),
    CONFIG(700),
    INFO(800),
    WARNING(900),
    SEVERE(1000),
    SHOUT(1200);
```
Example, user choose level `INFO(800)` -> only log with level >= INFO(800) can be displayed. Otherwise, we should not accept.